### PR TITLE
feat/upgrade node

### PR DESCRIPTION
**Pull request by chriskalmar**
- **update husky**
- **update husky**
- **update nvm**
- **update node to 22**
- **update deps**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,21 +24,21 @@
...
     "lint-staged": "lint-staged",
-    "prepare": "husky install",
+    "prepare": "husky",
     "release": "np --no-publish",
     "version": "./update-versions.sh",
     "dev": "ts-node src/bin",
     "start": "NODE_ENV=production node dist/bin.js"
   },
   "engines": {
-    "node": ">=18"
```
Comment (n2o1988) [inline package.json:37]: @chriskalmar Node 22 is quite recent. Is it really necessary to have that as a minimum version, in the `engines` section? 
Do you really use features from the newest node engine that aren't available in Node 18? 
Thanks.

**Review by n2o1988 — COMMENTED** [review]
Hey there, just leaving some thoughts as the latest version of `lost-pixel` forces us to drop this (quite useful) library due to the stricter NodeJS engine requirement.

File: package.json (hunk 1 above)
Comment (chriskalmar) [reply package.json:37]: I hear you @n2o1988. Node 22 was maybe a bit too soon 😅 
There are some dependencies that are sitting on at least Node 20.
Let me have a look into it, maybe Node 18 is still manageable (for the few more months it still is in maintenance).

Comment (chriskalmar) [reply package.json:37]: fixed: https://github.com/lost-pixel/lost-pixel/pull/428

**Pull request merged**